.github/workflows/python-publish.yml

```
@@ -81,25 +88,76 @@
+          echo "$CHANGELOG_CONTENT" >> $GITHUB_OUTPUT
+          echo "EOF" >> $GITHUB_OUTPUT
+
+      - name: Create GitHub Release
+        if: steps.version.outputs.has_changes == 'true'
+        uses: actions/create-release@v1
+        env:
+          GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
+        with:
+          tag_name: python-v${{ steps.version.outputs.new_version }}
+          release_name: Python Package v${{ steps.version.outputs.new_version }}
+          body: |
```

actionlint (1.7.7)

property "changelog" is not defined in object type {changelog_text: {conclusion: string; outcome: string; outputs: {string => string}}; version: {conclusion: string; outcome: string; outputs: {string => string}}}

(expression)